### DISCUSSION TITLE
Lost Users After Update

### DISCUSSION THREAD

**Discussion by ccigas** [blakeblackshear/frigate#13790] [Ask A Question]
Hi, I updated to the latest docker image last night and noticed my users were reset. I vaguely remember this when I first started with .14. I need to use port 5000 to set passwords and create my user again. 

Is this expected?  

Bonus question, is there anything on the roadmap for non admin users and camera selection per user? I want to be able to give my in laws a login and to only see their cameras, not mine. 

Thanks.

**Comment by NickM-27** [comment]
this means your docker config is incorrect and your database was deleted

**Reply by ccigas** [reply]
Fun thanks, I’ll work on that.
